app/components/Button/Button.css

```diff
@@ -58,37 +74,83 @@ html[data-theme='dark'] .button {
 }
 
 .dark {
-  background: var(--color-red-3);
-  color: var(--color-white);
-  border-color: var(--color-red-1);
+  background: var(--color-red-2);
+  border: none;
+  color: var(--color-white) !important;
```

Comment:
NO :no_good:
If you want to override other styles, use the "cascading" property of CSS.

Reply:
🙃